package.json

```diff
@@ -4,14 +4,15 @@
   "private": true,
   "main": "src/index.tsx",
   "dependencies": {
+    "@react-spring/web": "*",
```

Comment:
I highly suggest to not use the `*` pattern. Be explicit about the versions.

Reply:
Added the version instead of `*`.